TWLight/users/migrations/0031_auto_20181010_1744.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
I made no changes to the UserProfile model, and am not sure why I have an AlterField migration on the same

Reply:
You picked up a migration caused by languages being added by wiki translate. Go ahead and delete this file from your branch, as I have discovered that I need to go scoop those migrations out of the production branch and push them back to master so that we don't end up in migration hell.